packages/graphql/src/translate/translate-top-level-match.ts

```diff
@@ -20,47 +20,51 @@
 import type { AuthOperations, Context, GraphQLWhereArg } from "../types";
 import type { Node } from "../classes";
 import { createAuthAndParams } from "./create-auth-and-params";
+import { lowerFirst } from "../utils/lower-first";
 import Cypher from "@neo4j/cypher-builder";
 import { createWherePredicate } from "./where/create-where-predicate";
+import { SCORE_FIELD } from "../graphql/directives/fulltext";
 
 export function translateTopLevelMatch({
+    matchNode,
     node,
     context,
     varName,
     operation,
 }: {
+    matchNode?: Cypher.NamedNode;
```

Comment:
It is not a good idea to expect a `NamedNode`.

This function should not really care on wether or not a node is named or not, as it is the same to the queryBuilder and it will make our future refactors harder, as we are trying to really less on explicitly named variables in Cypher

Reply:
Also, are we really using this variable? I don't see where we pass this :sweat_smile:

Reply:
I've changed this to accept a `Node` not just a `NamedNode`. The variable itself is used a lot throughout `createMatchClause`. It was originally only assigned within `createMatchClause` but I needed to be able to reference the same variable within translate-read